Command/ExplainAdminCommand.php

```diff
@@ -89,11 +89,7 @@ public function execute(InputInterface $input, OutputInterface $output)
             $output->writeln(sprintf('  - % -25s  % -15s % -15s', $name, $fieldDescription->getType(), $fieldDescription->getTemplate()));
         }
 
-        if ($this->getContainer()->has('validator.validator_factory')) {
-            $metadata = $this->getContainer()->get('validator.validator_factory')->getMetadataFor($admin->getClass());
-        } else {
-            $metadata = $this->getContainer()->get('validator')->getMetadataFor($admin->getClass());
-        }
+        $metadata = $this->getContainer()->get('validator')->getMetadataFor($admin->getClass());
```

Comment:
Error:

#: php app/console sonata:admin:list
....cut...

[Symfony\Component\Debug\Exception\UndefinedMethodException]
  Attempted to call an undefined method named "getMetadataFor" of class "Symfony\Bundle\FrameworkBundle\Validator\ConstraintValidatorFactory".

Really: [ConstraintValidatorFactory.php](https://github.com/symfony/framework-bundle/blob/master/Validator/ConstraintValidatorFactory.php)